src/js/stores/UserLanguageStore.js

```diff
@@ -0,0 +1,26 @@
+import { EventEmitter } from "events";
+import Util from "../utils/Util";
+import UserSettingsStore from "./UserSettingsStore";
+import { SAVED_STATE_KEY } from "../constants/UserSettings";
+
+const getLanguagePreference = () =>
+  Util.findNestedPropertyInObject(
+    UserSettingsStore.getKey(SAVED_STATE_KEY),
+    "language"
+  ) || "en";
```

Comment:
Shouldn't we try to detect the browser language setting here? whitelisted by `constants/Languages` of course